### PR TITLE
Polish The Graph doc

### PR DESCRIPTION
**Pull request by meganskye**
remove reference to hosted service

### CODE REVIEW

**Comment by meganskye** [comment]
let's wait until we can publish zkatana subgraphs before we update this for the final time and merge it.

File: .gitignore [hunk 1]
```diff
@@ -27,3 +27,5 @@ yarn-error.log*
 !.yarn/sdks
 !.yarn/versions
 # .pnp.* # Comment for zero-install
+package.json
```
Comment (gluneau) [inline .gitignore:30]: This is odd, why git ignore the package list?

I understand, some may want to not store `package-lock.json`

**Review by gluneau — CHANGES_REQUESTED** [review]
fix git ignore

File: .gitignore (hunk 1 above)
Comment (meganskye) [reply .gitignore]: mistake on my part. fixed.

**Review by gluneau — APPROVED** [review]
LGTM

**Pull request merged**